examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -200,5 +222,35 @@ bool NativeStackTrace::error_occured() const {
   return error_occurred;
 }
 
+void NativeStackTrace::prune_dead_pid(uint32_t dead_pid) {
+  auto it = procSymbolsCache.find(dead_pid);
+  if (it != procSymbolsCache.end()) {
+    procSymbolsCache.erase(it);
+  }
+}
+
+void NativeStackTrace::enable_dso_reporting() {
+  NativeStackTrace::insert_dso_name = true;
+}
+
+ProcSyms* NativeStackTrace::get_proc_symbols(uint32_t pid) {
+  static struct bcc_symbol_option symbol_options = {
```

Comment:
Since it's not `const` - I think it's best to define it without `static`, so in case the function does modify it in any way, it will not affect subsequent calls.

Reply:
Sure.